docs/zh/Development/ContributeGuide.md

```diff
@@ -142,16 +142,16 @@ import -> Maven -> Existing Maven Projects
 
 
 
-## 常见编译错误
+# 常见编译错误
 
 1. 无法下载 thrift-* 等文件, 例如 `Could not get content
 org.apache.maven.wagon.TransferFailedException: Transfer failed for https://github.com/jt2594838/mvn-thrift-compiler/raw/master/thrift_0.12.0_0.13.0_linux.exe`
-
 这一般是网络问题，这时候需要手动下载上述文件：
 
-* 根据以下网址手动下载上述文件;
-  * https://github.com/jt2594838/mvn-thrift-compiler/blob/master/thrift_0.12.0_0.13.0_mac.exe
-  * https://github.com/jt2594838/mvn-thrift-compiler/raw/master/thrift_0.12.0_0.13.0_mac.exe
-* 将该文件拷贝到thrift/target/tools/目录下
-* 重新执行maven的编译命令
+  * 根据以下网址手动下载上述文件;
```

Comment:
When the thrift is upgraded to 0.14, the message will be obsolete？Do need to upgrade those descriptions in this PR？Of course, it's ok to upgrade the description until the thrift's upgraded PR is merged.

Reply:
Ok. Let me update in this PR.